packages/components/src/Overlay/Popover.tsx

```diff
@@ -35,22 +35,32 @@ const _Popover = forwardRef<HTMLDivElement, PopoverProps>(
     });
 
     const isSmallScreen = useSmallScreen();
+    const theme = useTheme();
+
     return (
       <>
         {isSmallScreen ? (
-          <Underlay variant="modal" open={open}>
```

Comment:
Is this still needed?

Reply:
what do you mean @sebald ? The `<Underlay variant="modal" />` yes because we decided to go with this and create a solution for the tray afterwards or not?

Reply:
I mean the open prop.